test/extensions/transport_sockets/tls/integration/ssl_integration_test.cc

```diff
@@ -352,6 +353,98 @@ TEST_P(SslCertficateIntegrationTest, ServerRsa) {
   checkStats();
 }
 
+// Server configured on SPIFFE certificate validation for mTLS
+// clientcert.pem's san is "spiffe://lyft.com/frontend-team" so it should be accepted.
+TEST_P(SslCertficateIntegrationTest, ServerRsaSPIFFEValidatorAccepted) {
+  auto typed_conf = new envoy::config::core::v3::TypedExtensionConfig();
```

Comment:
where is this released? you just need a `envoy::config::core::v3::TypedExtensionConfig` instead of a pointer.

Reply:
the pointer is passed to `validationContext`, and owned and freed there. That's why I created a raw pointer here to avoid double free.